### DISCUSSION TITLE
how to program the SYSCFG0 fuse on the avr128da28 ??

### DISCUSSION THREAD

**Discussion by manes6969** [avrdudes/avrdude#1926] [Q&A] [answered]
Hi ,
How can I program the syscfg0 fuse on the avr128da28 using my pickit5 on avrdude   ?
Any help welcome.
Regards,
Ronny

**Comment by MCUdude** [comment, marked as answer]
Did you try the docs?

if not you can try `avrdude -c pickit5_updi -p avr128da28 -U syscfg0:w:0xff:m`, where `0xff` is the fuse value.

**Comment by manes6969** [comment]
Hi Mcudude ,

As far as I could see , it was not in the docs using version 8.
I tried your answer and it works like a charm. Thanks a lot !
You've made my day with this.
Greetings Ronny

**Comment by MCUdude** [comment]
Grat to hear you got it working. There is also an interactive "fuse calculator" built into Avrdude.

You can invoke it using the `-t` flag:

```
$ avrdude -c pickit5_updi -pavr128da28 -t
avrdude> config
config wdtperiod=t_off # 0
config wdtwindow=t_off # 0
config bodsleep=bod_disabled # 0
config bodactive=bod_disabled # 0
config bodsampfreq=bod_128hz # 0
config bodlevel=bod_1v9 # 0
config clksel=oschf # 0
config eesave=eex_erased # 0
config rstpincfg=gpio # 0
config crcsel=crc16 # 0
config crcsrc=nocrc # 3
config sut=sut_0ms # 0
config codesize=0 # 0
config bootsize=0 # 0
config key=nolock # 0x5cc5c55c

avrdude> config clksel
config clksel=oschf # 0

avrdude> config clksel=
config clksel=oschf  # 0 (factory)
# conf clksel=osc32k # 1

avrdude> config eesave
config eesave=eex_erased # 0

avrdude> config eesave=
config eesave=eex_erased    # 0 (factory)
# conf eesave=eex_preserved # 1

avrdude> config eesave=1

avrdude> quit

Avrdude done.  Thank you.

```

Terminal mode (`-t`) has lots of different features. You can view them by typing `help`.
